### ISSUE TITLE
Acceptance Test AT3-1

### ISSUE THREAD

**Issue by yabdelkawy** [MaelSemler/mini-cap#147]
**Acceptance test:** 

AT3-1

**Feature:** 

FT3-1

**Feature description:** 

As a user I want to connect the Google calendar to the app. 

**Acceptance criteria:**

Given that I have successfully logged in with my credentials,
I can go to in the hamburger menu and select schedule
and I can choose to connect my Google calendar to the application,
Then, I should be able to see my Google calendar being sync into the schedule page.

**Previous evaluation:**
Not previously evaluated

**Result:**

To be filled in by product owner upon evaluation.

**Date:**

To be filled in by product owner upon evaluation.

**Signature:**

To be filled in by product owner upon evaluation.

**Comment by mkshiblu** [comment]
The floor plan not showing. Am I missing something?

![image](https://user-images.githubusercontent.com/15908043/77978339-d96fc900-72cf-11ea-85b4-6779b8008f19.png)

**Comment by MaelSemler** [comment]
Indeed, we have not finished implementing this feature, thus it is not in the apk provided.

**Comment by mkshiblu** [comment]
ACCEPTED on sprint 4
